### PR TITLE
[ADD] models category and team related with users

### PR DESCRIPTION
**Pull request by JorgeQuinteros**

### CODE REVIEW

**Comment by aitorbouzas** [comment]
Rebase

**Comment by aitorbouzas** [comment]
@JorgeQuinteros add a Many2many from teams to categories too.

**Comment by JorgeQuinteros** [comment]
I forgot the many2many, one minute

File: helpdesk/models/helpdesk_category.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from odoo import api, fields, models
+
+class HelpdeskCategory(models.Model):
+
+    _name = 'helpdesk.ticket.category'
+
```
Comment (aitorbouzas) [inline helpdesk/models/helpdesk_category.py:7]: Add a _description please.

File: helpdesk/models/helpdesk_team.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from odoo import api, fields, models
+
+
+class HelpdeskTeam(models.Model):
+
+    _name = 'helpdesk.ticket.team'
+
```
Comment (aitorbouzas) [inline helpdesk/models/helpdesk_team.py:8]: Add a description please.

**Review by aitorbouzas — CHANGES_REQUESTED** [review]

**Pull request merged**